api/db/migrations/20200625113402_add-column-email-to-organizations.js

```diff
@@ -0,0 +1,14 @@
+const TABLE_NAME = 'organizations';
```

Comment:
Attention avec #1565, cette branche aurait du partir, de la branche pix-843-add-column-email-to-organizations

Reply:
Ticket rebased suite à merge de PIX-843

Reply:
Le script d'ajout de la colonne a été mergé sur dev, j'ai donc pu rebase sur dev et il n'y a maintenant plus que le commit concernant cette PR.